### PR TITLE
Refactor  /app/pages/Profile/Sandboxes/index.js to /app/pages/Profile/Sandboxes/index.tsx 🔨 Refactor 🧠 Overmind Hacktoberfest

### PR DESCRIPTION
**Pull request by ganes1410**
## What kind of change does this PR introduce?

Refactor #2621 @Saeris  @christianalfoni 

## What is the current behavior?
It was using class component with js file, and cerebral for state management

## What is the new behavior?
It now uses tsx instead of js. And Overmind is used for state management with help of `useOvermind` hook

## What steps did you take to test this? This is required before we can merge.

1. `yarn lint`
2. `yarn test`
3. `yarn typecheck`

## Checklist

- [ ] Documentation
- [x] Testing 
- [x] Ready to be merged
      
- [ ] Added myself to contributors table
      

In this PR,I changed another file, because it was throwing a typescript error.

### CODE REVIEW

File: packages/app/src/app/pages/Profile/Sandboxes/index.tsx [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+import { Link } from 'react-router-dom';
+import { Button } from '@codesandbox/common/lib/components/Button';
+import { dashboardUrl } from '@codesandbox/common/lib/utils/url-generator';
+import { useOvermind } from 'app/overmind';
+import { SandboxList } from 'app/components/SandboxList';
+import { Navigation, Notice, NoSandboxes } from './elements';
+
+const PER_PAGE_COUNT = 15;
+
+interface iProps {
```
Comment (Saeris) [inline packages/app/src/app/pages/Profile/Sandboxes/index.tsx]:
```suggestion
interface ISandboxesProps {
```

File: packages/app/src/app/pages/Profile/Sandboxes/index.tsx [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+  source: string;
+  page: number;
+  baseUrl: string;
+}
+
+export const Sandboxes: React.FC<iProps> = ({ source, page, baseUrl }) => {
```
Comment (Saeris) [inline packages/app/src/app/pages/Profile/Sandboxes/index.tsx]:
```suggestion
export const Sandboxes: React.FC<ISandboxesProps> = ({ source, page, baseUrl }) => {
```

**Comment by ganes1410** [comment]
@Saeris I have added the required change. Let me know if you need me to make any more changes.

**Comment by SaraVieira** [comment]
Hey!

I am so sorry it took so long to review this from me but unfortatly this file is now being redesigned and will be merged soon so there is no need to merge this :(

I am so sorry and I hope you understand
You can see the new PR by Drake here: https://github.com/codesandbox/codesandbox-client/pull/3115

**Pull request closed without merging**